client/attest_test.go

```diff
@@ -62,7 +62,7 @@ func TestFetchIssuingCertificateSucceeds(t *testing.T) {
 
 	leafCert, _ := getTestCert(t, []string{"invalid.URL", ts.URL}, testCA, caKey)
 
-	cert, err := fetchIssuingCertificate(leafCert)
+	cert, err := fetchIssuingCertificate(&http.Client{}, leafCert)
```

Comment:
In this file we should probably have a Global variable called `localClient` that's just defined as:

```go
// The default client is fine as we are just talking to localhost
var localClient = http.DefaultClient
```

Just to make it clear that we don't need anything special.